backend/src/nodes/properties/inputs/pytorch_inputs.py

```diff
@@ -20,9 +29,46 @@ def __init__(
     ):
         super().__init__(input_type, label)
 
-    def enforce(self, value):
+    def enforce(self, value: PyTorchModel):
+        if torch is not None:
+            assert isinstance(value, torch.nn.Module), "Expected a PyTorch model."
+            assert isPyTorchModel(value), "Expected a supported PyTorch model."
+        return value
+
+
+class SrModelInput(ModelInput):
+    def __init__(
+        self,
+        label: str = "Model",
+        input_type: Union[
+            str, ExpressionJson
+        ] = "PyTorchModel { arch: invStrSet(PyTorchModel::FaceArchs) }",
```

Comment:
Woops. `ExpressionJson` already includes `str`, so this union is useless. Seems like I did this wrong when I added `ExpressionJson`.